jenkins/Jenkinsfile

```diff
@@ -126,45 +134,48 @@ pipeline {
+            description: "The name to give the custom Dataproc image and GCE snapshot. This will be appended with a hyphen and the short version of the commit hash (first 7 characters). This should be changed only for testing purposes, as this script relies on the name here matching the name in leo .conf files.")
         booleanParam(name: "USE_CUSTOM_IMAGE_IDENTIFIER", defaultValue: false)
         string(name: "CUSTOM_IMAGE_ID", defaultValue: "sampleUniqueID",
             description: "Specify along with USE_CUSTOM_IMAGE_IDENTIFIER to build the VM image with a specific name")
         string(name: "GOOGLE_PROJECT", defaultValue: "broad-dsp-gcr-public",
             description: "The google project to use")
-        string(name: "DATAPROC_IMAGE_BUCKET", defaultValue: "gs://leo-dataproc-image",
-            description: "The path for the bucket where the custom Dataproc images will be stored")
-        string(name: "GCE_IMAGE_BUCKET", defaultValue: "gs://leo-gce-image",
-            description: "The path for the bucket where the custom GCE images will be stored")
+        string(name: "DATAPROC_IMAGE_BUCKET", defaultValue: "gs://leo-dataproc-image-creation-logs",
+            description: "The path for the bucket where the custom Dataproc image generation logs will be stored")
+        string(name: "GCE_SNAPSHOT_BUCKET", defaultValue: "gs://leo-gce-snapshot-creation-logs",
+            description: "The path for the bucket where the custom GCE snapshot generation logs will be stored")
         string(name: "DATAPROC_VERSIONS", defaultValue: "1.2.79-debian9 1.4.51-debian10",
             description: "A custom image will be build for each of these dataproc versions")
         booleanParam(name: "USE_CUSTOM_DOCKER_IMAGE_VERSIONS", defaultValue: false,
             description: "Check if you want to specify custom image versions below")
-        string(name: "terra_jupyter_base", defaultValue: "0.0.13")
-        string(name: "terra_jupyter_hail", defaultValue: "0.0.19")
-        string(name: "terra_jupyter_r", defaultValue: "1.0.5")
-        string(name: "terra_jupyter_python", defaultValue: "0.0.15")
-        string(name: "terra_jupyter_bioconductor", defaultValue: "1.0.5")
-        string(name: "terra_jupyter_gatk", defaultValue: "1.0.5")
-        string(name: "terra_jupyter_aou", defaultValue: "1.0.14")
-        string(name: "anvil_rstudio_bioconductor", defaultValue: "0.0.8")
-        string(name: "welder_server", defaultValue: "59be71a",
+        string(name: "terra_jupyter_base", defaultValue: "0.0.19")
+        string(name: "terra_jupyter_hail", defaultValue: "0.1.2")
+        string(name: "terra_jupyter_r", defaultValue: "1.0.13")
+        string(name: "terra_jupyter_python", defaultValue: "0.1.1")
+        string(name: "terra_jupyter_bioconductor", defaultValue: "1.0.13")
+        string(name: "terra_jupyter_gatk", defaultValue: "1.1.1")
+        string(name: "terra_jupyter_aou", defaultValue: "1.0.23")
```

Comment:
Didn't pick the latest possible for `terra-jupyter-aou` as per a [Slack thread](https://terra-bio.slack.com/archives/C9C4KKU65/p1619034101059900) with @calbach.

Reply:
These are just the default params for the Jenkins job, and IMO don't add much value:

1. if `USE_CUSTOM_DOCKER_IMAGE_VERSIONS==false` (as is default), these values aren't used at all
2. if `USE_CUSTOM_DOCKER_IMAGE_VERSIONS==true`, then we are usually explicitly populating the versions anyway

So I'm wondering if we should just remove all these `defaultValue`s?

Reply:
@rtitle  do you mean mean remove the defaults and always read the versions from automation's reference.conf if no version specified?

Reply:
I see, some merging of the 2 modes.. that may be useful. As is today, it's either or: use all reference.conf versions, or all explicit versions.

Reply:
That's all good thinking. I've updated the PR to only remove the defaults for now.